lib/interceptor.js

```diff
@@ -195,6 +195,8 @@ Interceptor.prototype.reqheaderMatches = function reqheaderMatches(
 ) {
   const reqHeader = this.reqheaders[key]
   let header = options.headers[key]
+
+  // headers on the request should only be strings or arrays
```

Comment:
The comment is confusing me a little. Should it say that `header` can either be a string or an array of strings? In which case can it be an array of strings?

Reply:
agreed, it's confusing. I'll beef it up.